presto-thrift/src/test/java/io/prestosql/plugin/thrift/integration/TestThriftDistributedQueries.java

```diff
@@ -28,10 +28,4 @@ protected QueryRunner createQueryRunner()
     {
         return createThriftQueryRunner(3, 3, false, ImmutableMap.of());
     }
-
-    @Override
-    public void testAssignUniqueId()
```

Comment:
The problem with this test is that it stresses connector. It already was useful to find bugs in connectors implementation very often.

Reply:
How did this find bugs in connectors? If that's the case, we should add an explicit test to smoke tests. Ideally, anything here that tests connectors will move to smoke tests (which are there as connector tests), but that's a larger project.

Reply:
It scans multiple tables at the same time (25x union). If connector for example implements pooling, this test will show if the pooling doesn't close connections etc.

I think the test should be named differently, but it's valuable test.

Reply:
Also it exposes issues with any other shared resources within connector implementation that are not handling concurrency correctly.

Reply:
@kokosing @sopel39 let's copy the test do Distributed Queries. It can be outside of this PR

Reply:
@findepi Shouldn't it go in smoke tests so that it runs for all connectors?

Reply:
I think distributed queries is a better place. Smoke tests are not implemented for each connector I guess. Other than that I would merge smoke tests with Distributed queries, I don't follow the difference between them.

Reply:
Which connectors don't have smoke tests that have distributed queries? I couldn't find any.

Reply:
i confirm. every connector impl'ing distributed queries test also impls smoke test.
some (those read-only) connectors implement smoke test only.

(can be that a connector doesn't impl any, i didn't check)